app/src/main/java/edu/akelael/comics/presenter/ComicListPresenter.java

```diff
@@ -0,0 +1,35 @@
+package edu.akelael.comics.presenter;
+
+import java.util.List;
+
+import edu.akelael.comics.model.ComicListModel;
+import edu.akelael.comics.model.IComicListModel;
+import edu.akelael.comics.model.IComicListModelCallbacks;
+import edu.akelael.comics.model.Marvel;
+import edu.akelael.comics.view.IComicListActivity;
+
+public class ComicListPresenter implements IComicListPresenter, IComicListModelCallbacks {
```

Comment:
I really like your idea...! 
Can you help me? Because I can't put the idea in code... 
This case is like this: VIEW needs COMICS, so PRESENTER tells MODEL "get me the comics". That is an async method.
When data is retrieved from server... MODEL has to say to PRESENTER "ok here is the data"? How without a callback? Sorry, I can't get it.... :(